pkg/route/ingress/ingress.go

```diff
@@ -577,6 +570,36 @@ func hasIngressOwnerRef(owners []metav1.OwnerReference) (string, bool) {
 	return "", false
 }
 
+func (c *Controller) isIngressManaged(ingress *networkingv1.Ingress) (bool, error) {
```

Comment:
Nit: drop the `is` and call it `ingressManaged()` for consistency.

Reply:
Done.